### PR TITLE
added slider to xml

### PR DESCRIPTION
**Pull request by sschuraytz**
trying to figure out how to connect the seekbar to its java class so I can make changes based on updates to the seekbar

### CODE REVIEW

**Comment by sschuraytz** [comment]
The slider for the drawing tool width works now. Should I merge it to master?

File: app/src/main/res/layout/activity_main.xml [hunk 1]
```diff
@@ -1,16 +1,39 @@
...
-        android:id="@+id/canvas"
+    <LinearLayout
         android:layout_width="match_parent"
-        android:layout_height="wrap_content" />
+        android:layout_height="wrap_content"
+        android:orientation="horizontal">
 
-</LinearLayout>
\ No newline at end of file
+        <com.sschuraytz.charcoaldrawing.DrawingView
```
Comment (andrewoid) [inline app/src/main/res/layout/activity_main.xml]: Both the DrawingView and the SeekBar should be able to go into the same RelativeLayout (main_layout) without having to be wrapped in a RelativeLayout/LinearLayout

File: app/src/main/java/com/sschuraytz/charcoaldrawing/MainActivity.java [hunk 2]
```diff
@@ -2,13 +2,47 @@
...
 import android.os.Bundle;
+import android.text.Layout;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.widget.SeekBar;
+import android.widget.Toast;
 
 public class MainActivity extends AppCompatActivity {
 
+    public SeekBar drawingThickness;
```
Comment (andrewoid) [inline app/src/main/java/com/sschuraytz/charcoaldrawing/MainActivity.java]: This should be `private`

File: app/src/main/res/layout/activity_main.xml (hunk 1 above)
Comment (sschuraytz) [reply app/src/main/res/layout/activity_main.xml]: Good point. It involved tweaking some of the other layout settings for the SeekBar, but it's definitely cleaner code.

**Review by andrewoid — APPROVED** [review]

**Pull request merged**